.gitignore

```diff
@@ -82,3 +86,13 @@ gen-external-apklibs
 # Python Byte-compiled / optimized / DLL files
 __pycache__/
 *.py[cod]
+venv/
+
+
+# UI test artifacts 
+.firebase_token*
+results/
+test_artifacts/
+google-cloud-sdk/
+*.jar
+*.gz
```

Comment:
nit: If possible we should try to define (partially) the path here. We recently wanted to add a jar to AC and this line bit us. :)

Reply:
@pocmo done.  Shall I also go back and change this in r-b and a-c?

Reply:
Yeah, let's do that :)